src/mame/jpm/pluto6.cpp

```diff
@@ -1,107 +1,723 @@
+P10: LEDs - Unsurprisingly, controls LEDs. Can support 256 LEDs OR 32 7-Segment displays.
+P11: Lamp sourcees - Part of the matrixed 256 lamp system. 
+	Manual says this is 48v, but seems to be either 12v or 48v depending on needs.
+P12: Reels - Connects to 24 of the open-drain outputs, some of the lamp signals and 6 inputs.
+P13 + P14: Expose the remaining TTL inputs and open-drain outputs.
+P15: Aux Outputs - 6 TTL outputs. Connects to the VFDs on the development board.
+P16: Multiplex Expansion - Can be used to expand the system to 512 lights and 512 LEDs. Insane.
+P17: IO Bus Expansion
+P18: Main power input for +5v, +/-12v and optionally +48v. Has a power supply ready signal input.
+P19: Speaker output connector from audio amp. (4-8ohm 15W)
+P20: ccTalk/HI2 dual channel connector.
+P21: RS232 A/C/D connector.
+P22: BACTA Dataport connector, a.k.a RS232 B.
+P23: RS485/TTL connector.
+P24: VGA Connector for EXP0.
+P25: Either a box header connecting to EXP1 (but not video), or VGA for EXP1.
+P26: Labelled as Ethernet on development board, but manual says it was for future expansion.
+P27: I2C connector
+SW1+2: Software configurable DIP switches.
+SW3: Software configurable switch.
+
+* Boot process *
+On power up, the CPU is held in reset by the glue logic. Meanwhile the PIC18 boots and uses the +5v LED as a heartbeat.
+The PIC18 uploads the FPGA bitstream before releasing the reset line on the CPU. The CPU then boots and runs code
+from the flash chip. This usually involves running through a security check and then loading either an S19 program from
+the CompactFlash, or from the add-on ROM board.
+
+* Security *
+TODO: Expand as more is learnt.
+
+The security is handled by at least three parts, the FPGA, the CPLD and the PIC18.
+he PIC18 loads the FPGA with a bitstream that is customised by Heber for each manufacturer. 
+What this entails isn't yet known, but it is known the development board has most/all of this removed.
+This means that the PIC18 is essentially a dongle for each game and needs to be dumped. Whilst it contains the bitstream,
+its also probably stored in an encrypted for and decrypted within the Heber CPLD/ASIC, as the FPGA bitstream lines connect 
+to this part, not the PIC18.
+
+GPU/Dual GPU: Commented out due to errors and there is not a currently suitable GPU core for the Pluto 6.
+
+* Game Notes *
+pl6_cm: 	Hybrid Pluto 6 + PC setup. They are connected over RS232. Pluto 6 is probably handling IO and security here.
+			Uses a Intel Celeron (2.1Ghz) PC with a Protech PROX3770 motherboard. Game runs on Windows XP.
+pl6_dev: 	Heber's official development kit for the Pluto 6. Only ships with a development BIOS and an SDK.
+			This "game" requires a bootable (FAT32 with Active partition) CHD, else it does nothing visually.
+			The CHD in this driver is an image of the CF card that was supplied by Heber in the SDK for demo purposes.
+			Security and FPGA encryption is disabled on this model.
+
```

Comment:
Can't find anything about the Protech PROX3770 beyond some bare listing. The earlier PROX3680 seems to be running on Intel chipset (ICH2 + Socket 370), still no BIOS tho as canon for industrial boards. 
If `pl6_demo` is supposed to be the x86 BIOS then it should go as a separate driver regardless because it's very out of scope to what's here (and, eventually, connect thru serial as two MAME instances).